src/openfermion/measurements/_data_containers.py

```diff
@@ -0,0 +1,99 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""
+Classes to store experimental data for postprocessing.
+"""
+
+
+class QPERoundData(object):
+    """ Results from a single round in a QPE experiment.
+    Attributes:
+        nrot (int): the number of rotations performed in this round
+        finrot (float): the final rotation performed on the ancilla qubit
+            in this round.
+        msmt (bool): the measurement observed in this round
+        tmsmt (bool): if the ancilla is not reset between rounds, the
+            'true' measurement (in this case msmt is calculated as the
+            difference between the tmsmt of different rounds).
```

Comment:
I find the explanation of `true_measurement` to be confusing. Please document here what a QPE experiment is. If you did that somewhere else, you can move it to here.

Reply:
I've updated with a fairly lengthy description, hope it makes sense.